Tests/Blazorise.Benchmark/Program.cs

```diff
@@ -4,29 +4,30 @@
 using System.Linq;
 using BenchmarkDotNet.Attributes;
 using BenchmarkDotNet.Running;
+using Blazorise.Benchmark.DataGrid;
 using Blazorise.Extensions;
 #endregion
 
 namespace Blazorise.Benchmark
 {
     public class Program
     {
-        static void Main( string[] args )
+        private static void Main( string[] args )
         {
-            Console.WriteLine( "Hello World!" );
-
-            _ = BenchmarkRunner.Run<SequenceEquals>();
+            //_ = BenchmarkRunner.Run<SequenceEquals>();
+            _ = BenchmarkRunner.Run<ReflectionBenchmark>();
         }
 
         public class SequenceEquals
         {
             private List<string> simpleList1 = new() { "A", "B", "C" };
             private List<string> simpleList2 = new() { "A", "B", "C" };
+
             [Benchmark]
             public bool SequenceEqual() => simpleList1.SequenceEqual( simpleList2 );
 
             [Benchmark]
             public bool ArraySequenceEqual() => simpleList1.AreEqual( simpleList2 );
         }
     }
-}
+}
```

Comment:
I get a lot of _noise_ in PR because of this line ending. Did you use some kind of auto-format?

Reply:
I ran a tool, would you prefer that I revert?

Reply:
Yeah, its easier to review pr when there are less changes